extension-style-app/src/androidTest/java/com/mapbox/maps/testapp/style/sources/generated/GeoJsonSourceTest.kt

```diff
@@ -303,7 +343,8 @@ class GeoJsonSourceTest : BaseStyleTest() {
 
   @Test
   fun featureTest() {
-    val latch = CountDownLatch(1)
+    val latch = CountDownLatch(2)
```

Comment:
Do we know why `2` is needed now?

Reply:
we now set data with `style.addStyleSource` with empty values (becaues we allow to set GeoJsonSource with empty source data) and then apply the feature, geometry, data, featureCollection using `setStyleGeoJSONSourceData`, so 2 latch to verify both calls.